ResearchKit/Common/ORKAnswerFormat_Internal.h

```diff
@@ -135,6 +135,10 @@ ORK_DESIGNATE_CODING_AND_SERIALIZATION_INITIALIZERS(ORKTimeIntervalAnswerFormat)
 - (UIImage *)minimumImage;
 - (NSArray<ORKTextChoice *> *)textChoices;
 
+@optional
```

Comment:
Maybe this is error prone, as these two methods feel like they should be required for an `ORKTextScaleAnswerFormat`. How about making a `ORKTextScaleAnswerFormatProvider` protocol which inherits from `ORKScaleAnswerFormatProvider`, and then making `ORKTextScaleAnswerFormat` conform to the new protocol.